---
layout: aboutme
title: Research Interests | Ian Mellis
---

# <center> Research Interests </center>

## <center> Host-pathogen interactions </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/host-pathogen_img.png" style="width:600px;" />
</p>

In my postdoc with Dr. David Ho, I am pursuing several projects related to adaptive immune responses to viruses. In particular, I am interested in 1) better understanding neutralizing antibody responses to a variety of viruses; and 2) developing treatments (or vaccines) for viral infections using strategies informed by models of antigenic evolution and viral gene regulation. We have published or posted a number of studies about [SARS-CoV-2 evolution](BA_2_75.html), [COVID-19](XBB_MV.html) [vaccines](bivalent3mos.html), and associated adaptive immune responses, with more on the way. Currently, I am also exploring new projects on topics including correlates of protection from SARS-CoV-2, latent HIV reservoirs, and treatments for other viruses of pandemic potential. Ultimately, I hope to apply insights from these studies to the field of gene and cell therapies, as well.

<!-- ## <center> Manipulation of cellular identity </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/ipsc_cm_forsite.jpg" style="width:400px;" />
</p> -->
<!-- Cells display complex and important phenotypes depending on their cellular identity. For both basic and translational purposes we would like to be able to manipulate cellular identity on demand. For example, in basic research the advent of directed differentiation of cardiac myocytes (A, above) from human induced pluripotent stem cells (iPSC) has made possible experiments that previously would have required precious human tissue samples. Translationally, cellular reprogramming and transdifferentiation (directing cells to progenitor or other differentiated types, respectively; B, above) form the basis of many regenerative therapies under investigation.  -->
<!-- For more information, please see my most recent [preprint](p3.html). In the future I hope to develop additional transdifferentiation and reprogramming protocols for different starting and ending cell types. In particular, I am interested in high-fidelity and high-efficiency transdifferentiation to cardiac myocyte and motor neuron identities with an eye towards developing regenerative therapies for cardiac fibrosis, neurodegenerative disorders, and central nervous system injury. -->

## <center> Gene regulation and genetic engineering </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/inofish_forsite.jpg" style="width:600px;" />
</p>

In addition to the experimental and analytical frameworks above I am also interested in studying gene regulation and developing related new experimental methods to answer challenging research questions. In the first part of my PhD thesis we invented a single-molecule RNA FISH-based tool, [inoFISH](inoFISH.html), for visualizing RNA editing with sub-single-cell resolution. I further contributed to the development of other RNA FISH-based methods for [amplifying RNA FISH](clampFISH.html) signal and for visualizing [allelic expression](TissueSNPFISHbursting.html) in mouse tissues. Prior to graduate school I also contributed to the design and implementation of a [conditional knockout](COIN.html) module for genome engineering. More recently, we published a study about [transcriptional adaptation](GRN_NITC.html), a gene regulatory phenomenon that allows organisms to compensate for mutations. I am pursuing projects related to leveraging transcriptional adaptation for engineering gene upregulation in my postdoctoral research.

## <center> Systems biology of cellular identity </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/pillars-springs_forsite.jpg" style="width:400px;" />
</p>
I am interested in developing quantitative models for thinking about how and why cells have cellular identity (a.k.a. cell type) at a molecular level and from an evolutionary perspective. In my PhD thesis we identified gene regulatory trends associated with cellular identity maintenance. Relatedly, I am interested in designing protocols for reprogramming and transdifferentiation (directing cells to progenitor or other differentiated types, respectively) based on new models of the molecular and genetic basis of different cellular identities. We made progress toward this goal for more efficient fibroblast-to-iPSC reprogramming by suppressing genes that may be needed to maintain fibroblast identity. For more information, please see our [paper](p3.html). Currently, I am independently pursuing analytical projects about 1) how cells make tradeoffs in single-cell gene expression patterns to balance regulation of their diverse functions and 2) how to visualize cellular identity landscapes based on models of transcription factor-regulated cellular identity. 

<!-- ## Future Directions -->

