---
layout: aboutme
title: Research Interests | Ian Mellis
---

# <center> Research Interests </center>

## <center> Systems biology of cellular identity </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/pillars-springs_forsite.jpg" style="width:400px;" />
</p>
I am interested in developing quantitative models for thinking about how and why cells have cellular identity (a.k.a. cell type) at a molecular level and from an evolutionary perspective. 
<!-- In my thesis we developed a model for identifying transcription factors that maintain cellular identity. We showed that transcription factors whose expression levels change most frequently in response to a broad panel of perturbations <i>in vitro</i> are often important for cell identity maintenance <i>in vitro</i>.--> Currently I am continuing my thesis-related work on gene regulatory trends associated with cellular identity maintenance. For more information, please see my most recent [preprint](p3.html). I am also independently pursuing analytical projects about 1) how cells make tradeoffs in single-cell gene expression patterns to balance regulation of their diverse functions, and 2) how to visualize cellular identity landscapes based on models of transcription factor-regulated cellular identity. 

## <center> Manipulation of cellular identity </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/ipsc_cm_forsite.jpg" style="width:400px;" />
</p>

<!-- Cells display complex and important phenotypes depending on their cellular identity. For both basic and translational purposes we would like to be able to manipulate cellular identity on demand. For example, in basic research the advent of directed differentiation of cardiac myocytes (A, above) from human induced pluripotent stem cells (iPSC) has made possible experiments that previously would have required precious human tissue samples. Translationally, cellular reprogramming and transdifferentiation (directing cells to progenitor or other differentiated types, respectively; B, above) form the basis of many regenerative therapies under investigation.  -->
I am interested in designing protocols for reprogramming and transdifferentiation (directing cells to progenitor or other differentiated types, respectively) based on new models of the molecular and genetic basis of different cellular identities. In my thesis we made progress toward this goal for more efficient fibroblast-to-iPSC reprogramming by suppressing genes that may be needed to maintain fibroblast identity. For more information, please see my most recent [preprint](p3.html). In the future I hope to develop additional transdifferentiation and reprogramming protocols for different starting and ending cell types. In particular, I am interested in high-fidelity and high-efficiency transdifferentiation to cardiac myocyte and motor neuron identities with an eye towards developing regenerative therapies for cardiac fibrosis, neurodegenerative disorders, and central nervous system injury.

## <center> Tools for studying gene regulation </center> 

<p align="center">
<img src="{{ site.baseurl }}static/img/inofish_forsite.jpg" style="width:600px;" />
</p>

In addition to the experimental and analytical frameworks above I am also interested in developing accessible new experimental methods to answer challenging research questions, especially related to gene regulation. In the first part of my PhD thesis we invented a single-molecule RNA FISH-based tool, [inoFISH](inoFISH.html), for visualizing RNA editing with sub-single-cell resolution. I further contributed to the development of other RNA FISH-based methods for [amplifying RNA FISH](clampFISH.html) signal and for visualizing [allelic expression](TissueSNPFISHbursting.html) in mouse tissues. Prior to graduate school I also contributed to the design and implementation of a [conditional knockout](COIN.html) module for genome engineering.


<!-- ## Future Directions -->

